priv/templates/phx.gen.auth/confirmation_controller_test.exs

```diff
@@ -69,15 +69,21 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
       refute get_session(conn, :<%= schema.singular %>_token)
       assert Repo.all(<%= inspect context.alias %>.<%= inspect schema.alias %>Token) == []
 
+      # When not logged in
       conn = get(conn, Routes.<%= schema.route_helper %>_confirmation_path(conn, :confirm, token))
       assert redirected_to(conn) == "/"
-      assert get_flash(conn, :error) =~ "Confirmation link is invalid or it has expired"
+      assert get_flash(conn, :error) =~ "Account confirmation link is invalid or it has expired"
+
+      # When logged in
+      conn = build_conn() |> login_<%= schema.singular %>(<%= schema.singular %>) |> get(Routes.<%= schema.route_helper %>_confirmation_path(conn, :confirm, token))
```

Comment:
```suggestion
      conn = build_conn() |> log_in_<%= schema.singular %>(<%= schema.singular %>) |> get(Routes.<%= schema.route_helper %>_confirmation_path(conn, :confirm, token))
```

I believe this will fix the following compilation error

```
== Compilation error in file test/live_app_web/controllers/user_confirmation_controller_test.exs ==
     ** (CompileError) test/live_app_web/controllers/user_confirmation_controller_test.exs:78: undefined function login_user/2
         (elixir 1.10.4) src/elixir_locals.erl:114: anonymous fn/3 in :elixir_locals.ensure_no_undefined_local/3
         (stdlib 3.13) erl_eval.erl:680: :erl_eval.do_apply/6
         (elixir 1.10.4) lib/kernel/parallel_compiler.ex:396: Kernel.ParallelCompiler.require_file/2
         (elixir 1.10.4) lib/kernel/parallel_compiler.ex:306: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/7
```

Also I saw there's a formatter error on this file. I'd suggest generating the code, running the formatter, seeing what changes and making those updates here.

Reply:
Hi @aaronrenner .Thanks for pointing out the error. I also ran the formatter as you suggested and applied the changes to the template.